overlord/snapstate/snapmgr.go

```diff
@@ -411,6 +442,25 @@ func (snapst *SnapState) LocalRevision() snap.Revision {
 	return local
 }
 
+// LocalComponentRevision returns the "latest" local revision for the compName
+// component. Local revisions start at -1 and are counted down. 0 will be
+// returned if no local revision for the component is found.
+func (snapst *SnapState) LocalComponentRevision(compName string) snap.Revision {
```

Comment:
same question

Reply:
I have made this just a wrapper of a method in the sequence package now. To make things coherent I've done the same to `LocalRevision()`, as I think that we should reduce/remove usage of `SnapSequence.SideInfos()` as that method feels a bit like cheating, but let me know if this makes sense.